config/config/sc-config.yaml

```diff
@@ -1240,3 +1240,35 @@ networkPolicies:
         - "set-me"
       ports:
         - 443
+
+# Tekton Configuration
+tekton:
+  bootstraped: false
+  components:
+    pipelines:
+      enabled: false
+    dashboard:
+      enabled: false
+    triggers:
+      enabled: false
+    chains:
+      enabled: false
+  serviceAccount:
+    name: tekton-service
+    clusterAdmin: false
```

Comment:
Should we keep this here or move this together with the upgrade pipelines?
I feel like it could interfere the upgrade pipeline if it is here.

Reply:
Hmm I'm not sure I follow how you mean it would interfere, don't we want to use the same SA?